jobs/build-arch.Jenkinsfile

```diff
@@ -1,14 +1,13 @@
 import org.yaml.snakeyaml.Yaml;
 
-def pipeutils, streams, official, repo, gp
+def pipeutils, streams, official, uploading
 def src_config_url, src_config_ref, s3_bucket
+def session
```

Comment:
Does this need to be global? Can we declare it where we define it instead?

Reply:
I think you are right. Will fixe